config/deploy.rb

```diff
@@ -1,12 +1,12 @@
 # config/deploy.rb file
 require 'bundler/capistrano'
 
-set :application, "scholarship-application-app"
+set :application, "longshot"
 set :deploy_to, ENV["DEPLOY_PATH"]
 server  ENV["SERVER_NAME"], :app, :web
 
-set :user, "ubuntu"
-set :group, "ubuntu"
+set :user, "dosomething"
```

Comment:
Well, if we ever move to a new, exotic OS, we'll be glad everything's not called `ubuntu`. So 👍 and 🍕